Lib/test/test_crossinterp.py

```diff
@@ -111,6 +380,78 @@ class _GetXIDataTests(unittest.TestCase):
 
     MODE = None
 
+    def assert_functions_equal(self, func1, func2):
+        assert type(func1) is types.FunctionType, repr(func1)
+        assert type(func2) is types.FunctionType, repr(func2)
+        self.assertEqual(func1.__name__, func2.__name__)
+        self.assertEqual(func1.__code__, func2.__code__)
+        self.assertEqual(func1.__defaults__, func2.__defaults__)
+        self.assertEqual(func1.__kwdefaults__, func2.__kwdefaults__)
+        # We don't worry about __globals__ for now.
+
+    def assert_exc_args_equal(self, exc1, exc2):
+        args1 = exc1.args
+        args2 = exc2.args
+        if isinstance(exc1, ExceptionGroup):
+            self.assertIs(type(args1), type(args2))
+            self.assertEqual(len(args1), 2)
+            self.assertEqual(len(args1), len(args2))
+            self.assertEqual(args1[0], args2[0])
+            group1 = args1[1]
+            group2 = args2[1]
+            self.assertEqual(len(group1), len(group2))
+            for grouped1, grouped2 in zip(group1, group2):
+                # Currently the "extra" attrs are not preserved
+                # (via __reduce__).
+                self.assertIs(type(exc1), type(exc2))
+                #self.assert_exc_equal(grouped1, grouped2)
```

Comment:
Is this still intentionally commented out?  The assertion passed for me.

Reply:
fixed